API/ExecutorInterface.php

```diff
@@ -27,4 +27,13 @@ public function supportedTypes();
      * @throws \Exception
      */
     public function execute(MigrationStep $step);
+
+    /**
+     * Generate a update template
+     *
+     * @param string $identifier
+     * @return string Yaml dump
+     * @throws \Exception
+     */
+    public function generateTemplate($identifier);
```

Comment:
...or maybe we could make support for generateTemplate (pr whatever we call it) optional, and add instead a 'canGenerateMigration()' method (or interface?). This way we could, in the Command,  scan all known executors and see what migrations they can generate, so that we could validate the 'type' parameter passed in by the user...